lib/single_cov.rb

```diff
@@ -4,6 +4,7 @@ module SingleCov
   MAX_OUTPUT = 40
   RAILS_APP_FOLDERS = ["models", "serializers", "helpers", "controllers", "mailers", "views", "jobs", "channels"]
   UNCOVERED_COMMENT_MARKER = /#.*uncovered/
+  PREFIXES_TO_IGNORE = []
```

Comment:
so this is basically `prefixes_to_not_prefix_with_lib` or something like that right ?